src/core/iomgr/tcp_server_windows.c

```diff
@@ -242,6 +242,9 @@ static void start_accept(server_port *port) {
 
 /* Event manager callback when reads are ready. */
 static void on_accept(void *arg, int from_iocp) {
+  DWORD transfered_bytes = 0;
+  DWORD flags;
+  BOOL wsa_success;
```

Comment:
Ah, it looks like I've duplicated what was already done.
c:\jenkins\workspace\grpc_pull_requests@2\config\dbg\language\csharp\platform\windows\src\core\iomgr\tcp_server_windows.c(256): error C2086: 'DWORD transfered_bytes' : redefinition